dix/req.c

```diff
@@ -1791,7 +1849,10 @@ static uint32_t dix_rop_tgt_iter_max(struct m0_dix_req    *req,
 		 */
 		return m0_dix_liter_P(iter);
 	else
-		return m0_dix_liter_N(iter) + 2 * m0_dix_liter_K(iter);
+		/* Skip spares when DTM0 is enabled */
+		return ENABLE_DTM0 ?
+			m0_dix_liter_N(iter) + m0_dix_liter_K(iter) :
+			m0_dix_liter_N(iter) + 2 * m0_dix_liter_K(iter);
```

Comment:
From now on, please don't assume that N+K+K configuration.
We should have N+k+S (S == 0 or S == K).

Please refer to #375

Reply:
Yes, but #375 is not merged yet and we don't have an explicit m0_dix_liter_S() function. Will change when #375 is in main.